.github/workflows/tests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: run the tests
         run: python3 tests.py 
       - name: verify type hints
-        run: mypy . --strict
+        run: mypy . --strict --enable-incomplete-feature=Unpack --install-types
```

Comment:
`--install-types` will cause mypy to run twice. It's better to just know and install the stub-only packages you need ahead of time. I've updated my original PR.